docs/docs/config/key-binding.md

```diff
@@ -63,3 +65,28 @@ atuin init fish | source
+| alt + 1 to alt + 9                        | Select item by the number located near it                                                                          |
+| ctrl + c / ctrl + d / ctrl + g / esc      | Return original                                                                                                    |
+| ctrl + ← / alt + b                        | Move the cursor to the previous word                                                                               |
+| ctrl + h / ctrl + b / ←                   | Move the cursor to the left                                                                                        |
+| ctrl + → / alt + f                        | Move the cursor to the next word                                                                                   |
+| ctrl + l / ctrl + f / →                   | Move the cursor to the right                                                                                       |
+| ctrl + a / home                           | Move the cursor to the start of the line                                                                           |
+| ctrl + e / end                            | Move the cursor to the end of the line                                                                             |
+| ctrl + backspace / ctrl + alt + backspace | Remove the previous word / remove the word just before the cursor                                                  |
+| ctrl + delete / ctrl + alt + delete       | Remove the next word or the word just after the cursor                                                             |
+| ctrl + w                                  | Remove the word before the cursor even if it spans across the word boundaries                                      |
+| ctrl + u                                  | Clear the current line                                                                                             |
+| ctrl + n / ctrl + j                       | Select the next item on the list                                                                                   |
+| ctrl + p / ctrl + k / up arrow            | Select the previous item on the list                                                                               |
+| page down                                 | Scroll search results one page down                                                                                |
+| page up                                   | Scroll search results one page up                                                                                  |
+| enter                                     | Select highlighted item                                                                                            |
+| ↓                                         | Return original or return query depending on settings if no entries are found, or select the next item on the list |
```

Comment:
Maybe we can duplicate the line here to be a bit simpler
```suggestion
| ↓ (with no entry selected)                | Return original or return query depending on settings                                                              |
| ↓                                         | Select the next item on the list                                                              |
```
(this last line can be merged with the `ctrl + n / ctrl + j` line)